src/service/vaults.ts

```diff
@@ -213,8 +213,17 @@ export const watchVaultFactory = (netconfigUrl: string) => {
           liquidationPadding,
         );
 
+      const debtLimit = current.DebtLimit.value;
+
+      // The real debt limit can actually never be reached, only 1 less than
```

Comment:
Hopefully we can kill this code after https://github.com/Agoric/agoric-sdk/issues/6970